docs/books/admin_guide/13-softwares.md

```diff
@@ -523,6 +523,168 @@ gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-rockyofficial # GPG public key path
+
+Transaction Summary
+========================================================================================================================================
+Install  2 Packages
+
+Total download size: 1.7 M
+Installed size: 6.1 M
+Is this ok [y/N]:
+```
+
+Answering "y" to the prompt will install everything you need to use postgresql version 12 as a client.
+
+### Module Removal and Reset
+
+After you install, you may decide that for whatever reason, you need a different version of the stream. The first step is to remove your packages. Using our example `postgresql` package again, we would do this with:
+
+```
+dnf remove postgresql
+```
+
+This will display similar output as the install procedure above, except it will be removing the package and all of its dependencies. Answer "y" to the prompt and hit enter to uninstall `postgresql`.
+
+Once this step is complete, you can issue the reset command for the module using:
+
+```
+dnf module reset postgresql
+```
+
+Which will give you output like this:
+
+```
+Dependencies resolved.
+========================================================================================================================================
+ Package                         Architecture                   Version                           Repository                       Size
+========================================================================================================================================
+Disabling module profiles:
+ postgresql/client                                                                                                                     
+Resetting modules:
+ postgresql                                                                                                                            
+
+Transaction Summary
+========================================================================================================================================
+
+Is this ok [y/N]:
+```
+
+Answering "y" to the prompt will then reset `postgresql` back to the default stream with the stream that we had enabled (12 in our example) no longer enabled:
+
+```
+Rocky Linux 8 - AppStream
+Name                       Stream                 Profiles                           Summary                                            
+postgresql                 9.6                    client, server [d]                 PostgreSQL server and client module                
+postgresql                 10 [d]                 client, server [d]                 PostgreSQL server and client module                
+postgresql                 12                     client, server [d]                 PostgreSQL server and client module                
+postgresql                 13                     client, server [d]                 PostgreSQL server and client module
+```
+
+Now you can use the default or enable another stream just like we did above.
```

Comment:
This section might need an addition of `dnf module switch-to` as this is somehow the better option of `dnf module reset x && dnf module enable x:x` now
https://dnf.readthedocs.io/en/latest/command_ref.html#module-command-label